packages/run-protocol/src/runStake/attestation.js

```diff
@@ -153,12 +167,12 @@ const makeAttestationIssuerKit = async (zcf, stakeBrand, lienBridge) => {
     await E(lienBridge).setLiened(address, lienedPre, lienedPost);
     // COMMIT. Burn the escrowed attestation payment and exit the seat
     amountByAddress.set(address, lienedPost);
-    zcfMint.burnLosses(harden({ Attestation: attestationAmount }), seat);
+    attMint.burnLosses(harden({ Attestation: attestationAmount }), seat);
     seat.exit();
   };
 
   /**
-   * Get liened amount synchronously.
+   * Get liened amount from the JS store, without using the lienBridge.
```

Comment:
please help reader understand the authority of this info. If the JS store and the lienBridge disagree about the amount, what should happen? (IIUC, that triggers death from confusion)

Reply:
ok, I'll add the info that this is mainly for testing to the code as well as this conversation.